send/config.c

```diff
@@ -80,6 +80,44 @@ static int smtp_auth_validator(const struct ConfigSet *cs, const struct ConfigDe
   return CSR_SUCCESS;
 }
 
+/**
+ * sendmail_validator - Validate the "sendmail" config * variable - Implements ConfigDef::validator() - @ingroup cfg_def_validator
+ */
+static int sendmail_validator(const struct ConfigSet *cs, const struct ConfigDef *cdef,
+                              intptr_t value, struct Buffer *err)
+{
+  const char *valstr = (const char *)value;
+
+  /* Check that there is a value */
+  if (valstr == NULL || *valstr == '\0')
+  {
+    // L10N: This applies to the "$sendmail" and "$inews" config variables.
+    mutt_buffer_printf(err, _("Option %s must not be empty"), cdef->name);
+    return CSR_ERR_INVALID;
+  }
```

Comment:
We don't need this `if`...

`$inews` is allowed to be empty.
If it is, then NeoMutt will post directly to the server.

For `$sendmail`, we've got a better option: `DT_NOT_EMPTY` (see below)